### ISSUE TITLE
Use Public Projects Board as Roadmap

### ISSUE THREAD

**Issue by H0R5E** [SNL-WaterPower/WecOptTool#117]
Given the speed of change in this project, the README based roadmap is not being utilised. I still think there is some value in trying to structure the open tasks into some sort of order by release date and priority. I think that a public facing projects board ([like this]( https://github.com/openservicebrokerapi/servicebroker/projects/1)) would allow us to do this more effectively than we have done with the README based roadmap (or just utilising the issues page alone), and communicate our plans to the user community.

With this system we can create future releases (or any other useful metric) as columns and then assign them the relevant issues. The height of the issue in the column would then determine its priority in the burn down list. The only drawback is that access control to these boards is quite simplistic, so I would recommend having a single "task master" and then using team meetings to try and set the releases and priority of tasks, so that there are no surprises.

**Comment by H0R5E** [comment]
OK, my first try at this is now here: https://github.com/SNL-WaterPower/WecOptTool/projects/2

I would like to now take the roadmap out of the README if this works for the team.

**Comment by ssolson** [comment]
Yes, this is much better. I always found the Roadmap in the Readme very hard to follow and it needed constant updating. Unrelated (I am not sure where I read it this morning) but it "WaveBot" because controls/ robot.

**Comment by H0R5E** [comment]
> Unrelated (I am not sure where I read it this morning) but the device is called the "WaveBot" because controls/ robot.

This was in #123, which I have now fixed, thanks.

**Comment by ryancoe** [comment]
I also strongly prefer the project board to the README roadmap and support removal of the roadmap.
